### PR TITLE
Extract sub-command for revoking expired certificates

### PR DESCRIPTION
**Pull request by mfurtak**
This functionality was introduced in #70 as an optional flag for normal `cert` operation.

This change isolates this behavior into its own sub-action to reduce the likelihood that it would ever be triggered accidentally (since it is a destructive behavior).

### CODE REVIEW

File: lib/cert/runner.rb [hunk 1]
```diff
@@ -44,6 +41,25 @@ def run
       end
     end
 
+    # Command method for the :revoke_expired sub-command
+    def revoke_expired_certs!
+      login
```
Comment (KrauseFx) [inline lib/cert/runner.rb]: Should we also call `FastlaneCore::PrintTable.print_values(config: Cert.config, hide_keys: [:output_path], title: "Summary for cert #{Cert::VERSION}")` at this point?

File: lib/cert/runner.rb [hunk 2]
```diff
@@ -44,6 +41,25 @@ def run
...
+          UI.message "#{certificate.id} #{certificate.name} has expired, revoking..."
+          certificate.revoke!
+          revoke_count += 1
+        rescue => e
+          UI.error "An error occurred while revoking #{certificate.id} #{certificate.name}"
+          UI.error "#{e.message}\n#{e.backtrace.join("\n")}" if $verbose
+        end
+      end
+
+      UI.success "#{revoke_count} expired certificate#{'s' if revoke_count != 1} #{revoke_count == 1 ? 'has' : 'have'} been revoked! 👍"
```
Comment (KrauseFx) [inline lib/cert/runner.rb]: We have that across the code base, maybe we should add a helper method to fastlane_core in the long run.

Comment (KrauseFx) [inline lib/cert/runner.rb]: +1 for emoji usage here

**Comment by KrauseFx** [comment]
Looks great, however I didn't try running it :+1:

File: lib/cert/runner.rb (hunk 1 above)
Comment (KrauseFx) [inline lib/cert/runner.rb]: Oh because it just revokes certs we don't really use all parameters, right?

Comment (mfurtak) [inline lib/cert/runner.rb]: Yeah, it doesn't use all params, but I think I agree that it would still be better if a summary table was printed.

**Pull request merged**